framework/contrib/gtest/gtest/gtest.h

```diff
@@ -51,6 +51,16 @@
 #ifndef GTEST_INCLUDE_GTEST_GTEST_H_
 #define GTEST_INCLUDE_GTEST_GTEST_H_
 
+// This warning is a known issue since 2019 (https://github.com/google/googletest/issues/2224)
+// Google has yet to fix it, so we will direct clang > 9 to ignore the warnings.
+#ifdef __clang__
+#pragma clang diagnostic push
+#if (__clang_major__ > 9)
+// This was introduced in clang 10
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
+#endif // clang > 9
+#endif
```

Comment:
This is exactly what I was hoping to see. Thanks!

Reply:
Thanks for prompting the change! I didn't know about these pragmas until I dug into those libmesh headers a bit more! :D